node/node-pakiti2-json.py

```diff
@@ -37,9 +38,9 @@
     print(response.read())
 
   except:
-    print('[]')
+    print('Cannot Connect')
```

Comment:
`json_decode` can be parsed strings. In this case `json_decode` will equal `false`.